core/templates/dev/head/pages/exploration_player/audio_bar_directive.html

```diff
@@ -7,7 +7,7 @@
          uib-tooltip="<[!isAudioAvailableInCurrentLanguage() ? ('I18N_PLAYER_AUDIO_NOT_AVAILABLE_IN' | translate:{languageDescription:getCurrentAudioLanguageDescription()}) : '']>"
          tooltip-append-to-body="true" tooltip-placement="right">
       </i>
-      <div class="slider-section" ng-if="hasPressedPlayButtonAtLeastOnce">
+      <div class="slider-section" ng-if="hasPressedPlayButtonAtLeastOnce &&  !isAudioAutoGenerated">
```

Comment:
Probably you forgot the unnecessary double space here. Thanks!

Reply:
Oh! sorry for that.
Thanks for review @YashJipkate

Reply:
@import-keshav Anytime :)